app/src/main/res/values-de/strings.xml

```diff
@@ -248,6 +248,8 @@
     <string name="pref_desc_ui_article_textAlignment_justify">Streckt die Textzeilen, so dass jede Zeile die gleiche Breite hat (wie in Zeitungen)</string>
     <string name="pref_name_ui_readingSpeed">Lesegeschwindigkeit</string>
     <string name="pref_desc_ui_readingSpeed">Deine Lesegeschwindigkeit (gemessen in Wörtern pro Minute). Zum Berechnen der geschätzten Lesezeit genutzt.</string>
+    <string name="pref_name_ui_keepScreenOn">Bildschirm während Lesens anlassen</string>
+    <string name="pref_desc_ui_keepScreenOn">Lasse den Bildschirm dauerhaft an, solange du einen Artikel ließt. </string>
```

Comment:
kleiner Tippfehler: "liest" statt ließt

Reply:
Sollte ich einen neuen Commit auf den Branch pushen der den Typo ändert, oder besser einen rebase durchführen?

Reply:
Danke, einer neuer Commit in deinem Branch ist ausreichend.